### PR TITLE
ref(docintegration): Use BooleanField, limit charfield max len

### PR DESCRIPTION
**Pull request by ceorourke**
Future proof the `DocIntegration` model by using `BooleanField` instead of `NullBooleanField` (the latter is [deprecated](https://djangodoctor.medium.com/nullbooleanfield-is-dead-7d3fc286ed6e) in a future version of Django so it's not a today problem) as well as limit the max character length of the name field.

### CODE REVIEW

File: src/sentry/models/integration.py [hunk 1]
```diff
@@ -28,14 +28,14 @@ class DocIntegration(Model):
...
 
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=64)
     slug = models.CharField(max_length=64, unique=True)
     author = models.CharField(max_length=255)
     description = models.TextField()
     url = models.URLField()
     popularity = models.PositiveSmallIntegerField(null=True, default=1)
     # Quickly hide/show integration, used for logo uploads
-    is_draft = models.NullBooleanField(default=True)
```
Comment (ceorourke) [inline src/sentry/models/integration.py:38]: I know altering a column type can be dangerous, but there are no rows in this table yet.

**Comment by ceorourke** [comment]
This is the last of my migrations 😄 😄 (for now 😈 )

File: src/sentry/models/integration.py [hunk 2]
```diff
@@ -28,14 +28,14 @@ class DocIntegration(Model):
...
+    is_draft = models.BooleanField(null=True)
```
Comment (leeandher) [inline src/sentry/models/integration.py:38]: Is it possible to remove the `null=True` all together? It's pretty much required, and we can retain `default=True` as well

File: src/sentry/models/integration.py (hunk 1 above)
Comment (wedamija) [reply src/sentry/models/integration.py:38]: In this case it's fine, the type of the column just remains `bool`

**Review by wedamija — APPROVED** [review]
lgtm since this table is unused.

**Pull request merged**